src/fides/api/ops/api/v1/endpoints/privacy_experience_endpoints.py

```diff
@@ -81,56 +70,34 @@ def privacy_experience_list(
             PrivacyExperience.disabled.is_(False)
         )
     if region is not None:
-        experience_query = experience_query.filter(
-            PrivacyExperience.regions.contains([region])
-        )
+        experience_query = experience_query.filter(PrivacyExperience.region == region)
     if component is not None:
         experience_query = experience_query.filter(
             PrivacyExperience.component == component
         )
+    if has_config is True:
+        experience_query = experience_query.filter(
+            PrivacyExperience.experience_config_id.isnot(None)
+        )
+    if has_config is False:
+        experience_query = experience_query.filter(
+            PrivacyExperience.experience_config_id.is_(None)
+        )
 
     results: List[PrivacyExperience] = []
     for privacy_experience in experience_query.order_by(
         PrivacyExperience.created_at.desc()
     ):
         privacy_notices: List[
             PrivacyNotice
-        ] = privacy_experience.get_related_privacy_notices(db, region, show_disabled)
+        ] = privacy_experience.get_related_privacy_notices(db, show_disabled)
         privacy_experience.privacy_notices = privacy_notices
         if not (has_notices and not privacy_notices):
             results.append(privacy_experience)
 
     return fastapi_paginate(results, params=params)
 
 
-@router.post(
-    urls.PRIVACY_EXPERIENCE,
-    status_code=HTTP_201_CREATED,
-    response_model=List[PrivacyExperienceResponse],
-    dependencies=[
-        Security(verify_oauth_client, scopes=[scope_registry.PRIVACY_EXPERIENCE_CREATE])
-    ],
-)
-def privacy_experience_create(
-    *,
-    db: Session = Depends(deps.get_db),
-    bulk_experience_data: conlist(PrivacyExperienceCreate, max_items=50),  # type: ignore
-) -> List[PrivacyExperience]:
-    """
-    Bulk create Privacy Experiences. Returns related notices in the response.
-    """
-    logger.info("Creating privacy experiences")
-    experiences: List[PrivacyExperience] = []
-    for experience_data in bulk_experience_data:
-        experience = PrivacyExperience.create(
-            db, data=experience_data.dict(exclude_unset=True), check_name=False
-        )
-        # Temporarily stash the privacy notices on the experience for display
-        experience.privacy_notices = experience.get_related_privacy_notices(db)
-        experiences.append(experience)
-    return experiences
-
-
```

Comment:
You can no longer edit a PrivacyExperience directly.  Instead, add or update Privacy Notices and I make sure a PrivacyExperience exists in the background that matches those notices.  Alternatively, update the ExperienceConfigs to format the display, and I persist those same changes back to the PrivacyExperiences.

For example, customer creates an Advertising CA Privacy Notice displayed in an overlay.  I then create a CA banner overlay Experience for them.   The customer then creates a banner overlay PrivacyExperienceConfig and specifies CA as a region.  I then link that CA overlay Experience to this PrivacyExperience config via FK.   (another alternative is that I have a default banner overlay experience that I could link to the experience as soon as it was created, that the customer could then updated).